lib/adrian/directory_queue.rb

```diff
@@ -17,27 +17,27 @@ def self.create(options = {})
     # Note:
     # There is the possibility of an item being consumed by multiple processes when its still in the queue after its lock expires.
     # The reason for allowing this is:
-    #   1. It's much simpler than introducing a seperate monitoring process to handle lock expiry.
+    #   1. It's much simpler than introducing a separate monitoring process to handle lock expiry.
     #   2. This is an acceptable and rare event. e.g. it only happens when the process working on the item crashes without being able to release the lock
     def initialize(options = {})
       super
       @available_path = options.fetch(:path)
       @reserved_path  = options.fetch(:reserved_path, default_reserved_path)
       @logger         = options[:logger]
+      @touch_first    = options[:touch_first]
```

Comment:
I think this should use `fetch` and have a default value of `false` in case it is not passed in with the options.